public/app/features/dashboard-scene/scene/keyboardShortcuts.ts

```diff
@@ -102,13 +102,23 @@ export function setupKeyboardShortcuts(scene: DashboardScene) {
       handleZoomOut(scene);
     },
   });
+
   keybindings.addBinding({
     key: 'ctrl+z',
     onTrigger: () => {
       handleZoomOut(scene);
     },
   });
 
+  // Relative -> Absolute time range
+  keybindings.addBinding({
```

Comment:
It would be great to have E2E here :) 

Also, the linked ticket is a private one. Can we update the description to reflect the need for the PR for future context? 

Thank you!